### PR TITLE
Update mqtt adapter imports

### PR DESCRIPTION
**Pull request by nmarcetic**
Signed-off-by: Nikola Marcetic <n.marcetic86@gmail.com>

### What does this do?
It updates mqtt adapter imports and adds mproxy WebSocket init. 
### Which issue(s) does this PR fix/relate to?
NA
### List any changes that modify/break current functionality
NA
### Have you included tests for your changes?
NA
### Did you document any new/modified functionality?
NA
### Notes
NA

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: cmd/mqtt/main.go [hunk 1]
```diff
@@ -27,43 +29,71 @@ import (
...
+	// Things
+	defThingsURL     = "localhost:8181"
+	defThingsTimeout = "1" // in seconds
+	envThingsURL     = "MF_THINGS_URL"
+	envThingsTimeout = "MF_MQTT_ADAPTER_THINGS_TIMEOUT"
+	// Nats
+	defNatsURL = broker.DefaultURL
+	envNatsURL = "MF_NATS_URL"
+	// Jaeger
+
```
Comment (manuio) [inline cmd/mqtt/main.go]: rm this blank line

**Review by manuio — CHANGES_REQUESTED** [review]

File: mqtt/adapter.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func New(tc mainflux.ThingsServiceClient, pubs []mainflux.MessagePublisher, es r
 
 // AuthConnect is called on device connection,
 // prior forwarding to the MQTT broker
-func (e *Event) AuthConnect(c *mqtt.Client) error {
+func (e *Event) AuthConnect(c *session.Client) error {
```
Comment (manuio) [inline mqtt/adapter.go:59]: Fix `AuthConenct` inside this function

**Review by manuio — CHANGES_REQUESTED** [review]

File: cmd/mqtt/main.go [hunk 3]
```diff
@@ -118,6 +148,9 @@ func main() {
 	logger.Info(fmt.Sprintf("Starting MQTT proxy on port %s ", cfg.mqttPort))
```
Comment (manuio) [inline cmd/mqtt/main.go]: Rm the white space after `%s`

File: cmd/mqtt/main.go [hunk 4]
```diff
@@ -118,6 +148,9 @@ func main() {
...
 	go proxyMQTT(cfg, logger, evt, errs)
 
+	logger.Info(fmt.Sprintf("Starting MQTT over WS  proxy on port %s ", cfg.httpPort))
```
Comment (manuio) [inline cmd/mqtt/main.go]: Rm the white space after %s

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**